src/store/user/slice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface userState {
+  isLogged: boolean;
+  isAdmin: boolean;
+}
+
+const initialState: userState = {
+  isLogged: false,
+  isAdmin: false,
+};
+
+export const userSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {
+    signIn: (state, action: PayloadAction<boolean>) => {
+      state.isLogged = true;
+      state.isAdmin = action.payload;
+    },
+    signOut: state => {
+      state.isLogged = false;
+      state.isAdmin = false;
+    },
+  },
+});
+
+export const { signIn, signOut } = userSlice.actions;
+export const { reducer } = userSlice;
```

Comment:
Import signIn from actions
```suggestion

const { reducer } = slice;

const actions = {
  ...slice.actions,
  signIn,
};

export { reducer, actions };
```